test/packages/knowledge_base/manifest.yml

```diff
@@ -0,0 +1,27 @@
+format_version: 3.4.0
+name: knowledge_base
+title: Knowledge base test content package
+description: >
+  This package is a dummy example for packages containing a knowledge base entry.
+version: 0.1.0
+type: content
+source:
+  license: "Elastic-2.0"
+conditions:
+  kibana:
+    version: '^8.15.0'
+  elastic:
+    subscription: 'enterprise'
```

Comment:
Do knowledge bases require enterprise subscription? (out of curiosity, no need to do anything about this here)

Reply:
Yeah, those features are gated behind enterprise subscription for now (mostly because it's part of the AI assistant feature, which is also gated behind enterprise)